src/index.ts

```diff
@@ -2,56 +2,89 @@ import { join } from 'path'
 import { existsSync } from 'fs'
 import type { Plugin, ResolvedConfig } from 'vite'
 import { generateSW, injectManifest } from 'workbox-build'
-import { generateSWRegister, injectServiceWorker } from './html'
+import { generateSimpleSWRegister, injectServiceWorker } from './html'
+import { generateRegisterSW } from './modules'
 import { ResolvedVitePWAOptions, VitePWAOptions } from './types'
-import { resolveOptions } from './config'
-import { FILE_MANIFEST, FILE_SW_REGISTER } from './constants'
+import { resolveOptions } from './options'
+import { FILE_MANIFEST, FILE_SW_REGISTER, VIRTUAL_MODULES, VIRTUAL_MODULES_MAP } from './constants'
 
-export function VitePWA(userOptions: Partial<VitePWAOptions> = {}): Plugin {
+export { cachePreset } from './cache'
```

Comment:
remove it, it is not used: @antfu we can also remove `cache.ts` module

Reply:
No, I kept it for people who want to use the previous behavior where they can import via

```ts
import { cachePreset } from 'vite-plugin-pwa`
```

Reply:
ok, just add it again, I remove it from the module